### DISCUSSION TITLE
Can i copy-paste into the desktop provided by the eclipsevnc image?

### DISCUSSION THREAD

**Discussion by manumaan** [coder/coder#6156] [General]
Can i copy-paste into the desktop provided by the eclipsevnc image? Currently what i have seen is I cant. But i can paste into terminal and code-server.

**Comment by matifali** [comment]
This depends on the VNC image you used. Not related to Coder.
There seems a bug but it is fixed now,
See https://bugs.eclipse.org/bugs/show_bug.cgi?id=70077 for more discussion.
